G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.download/ShouldDownloadSheet_WhenOpenAndDownloadSheet.G1ANT

```diff
@@ -0,0 +1,13 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
+♥testValue = ‴G1ANT‴
+♥folderPath =♥environment⟦TEMP⟧\
+♥filePath = ♥folderPath + "SheetsTest.xlsx"
+
+googlesheet.open ♥FileID isshared false
+googlesheet.download ♥filePath type ‴xls‴ result ♥resoult
```

Comment:
A typo in `resoult`
The result of download is not verified (file.exists), instead you just test the result message, which IMHO is not as important as the fact of file download.